org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/IProjectImporter.java

```diff
@@ -24,6 +24,8 @@
 
 	boolean applies(IProgressMonitor monitor) throws OperationCanceledException, CoreException;
 
+	boolean applies(File folder) throws OperationCanceledException, CoreException;
```

Comment:
should provide a default implementation such as return false. This will avoid breaking other plugins that registered importer but not implemented the new interface.

Reply:
Fixed